templates/cluster-template.yaml

```diff
@@ -198,8 +201,5 @@ metadata:
 spec:
   allowedNamespaces: {}
   clientID: ${AZURE_CLIENT_ID}
-  clientSecret:
-    name: ${AZURE_CLUSTER_IDENTITY_SECRET_NAME}
-    namespace: ${AZURE_CLUSTER_IDENTITY_SECRET_NAMESPACE}
   tenantID: ${AZURE_TENANT_ID}
-  type: ServicePrincipal
+  type: WorkloadIdentity
```

Comment:
If we don't want to use SP at all in tests, it seems like we could remove the workload-identity-specific test and templates.

Reply:
agree